Assets/Scripts/ScriptableObjects/AI.meta

```diff
@@ -0,0 +1,10 @@
+fileFormatVersion: 2
```

Comment:
There are currently two Scriptable Object folders.

Merge `/Scripts/ScriptableObjects` and `Scripts/Scriptable Objects`